### ISSUE TITLE
Display beta versions

### ISSUE THREAD

**Issue by calixteman** [mozilla/missioncontrol#198]
Rather than to have only buildid, it would be nice to have b1, b2, ...
Maybe buildhub could help here:
https://github.com/mozilla-services/buildhub

**Comment by wlach** [comment]
Support for this was implemented in mozilla/telemetry-streaming#83 (the information is in the telemetry ping), but waiting on a new version of that to be deployed.

**Comment by wlach** [comment]
I have backported the necessary functionality to the production branch, hopefully that means we can deploy this enabling functionality soon (there are issues with deploying the master branch on our infra): mozilla/telemetry-streaming#96

**Comment by wlach** [comment]
wasn't sure whether to mark this as a bug or a feature, I guess we'll consider we'll consider it a feature for now, albeit a very important one
